docker-compose.monitor.yml

```diff
@@ -14,10 +14,6 @@ services:
             - mender
         depends_on:
             - mender-mongo
-        environment:
-            DEVICEMONITOR_MONGO_URL: "mongodb://mender-mongo"
-            DEVICEMONITOR_USERADM_URL: "http://mender-useradm"
-            DEVICEMONITOR_WORKFLOWS_URL: "http://mender-workflows-server"
```

Comment:
yes, these are bad, I fixed them in the feature branch, why aren't you targeting feature-monitor?

Reply:
Should I target that feature branch instead of master?

Reply:
I believe so. at least full integration test went into the feature branch.